layouts/shortcodes/docs/instrumentation/index-adopters.md

```diff
@@ -0,0 +1,15 @@
+{{ $lang := .Get 0 | humanize -}}
+{{ $data := $.Site.Data.ecosystem.adopters }}
+
+## Who's using OpenTelemetry {{ $lang }}?
+
+OpenTelemetry {{ $lang }} is in use by a number of organizations, including:
+
+{{- range $data }} 
+    {{- if in .components $lang }}
+- [{{ .name }}]({{ .url }}) 
+    {{- end }}
+{{- end }}
+
+To have your organisation listed, follow the
+[instructions given at the bottom of the Adopters page](/ecosystem/adopters/).
```

Comment:
Prettier is acting strange. I could only get it to work with `prettier-ignore`, not with the start/end directives. 

Anyhow, I've also include other changes in the suggestion below:

```suggestion
<!-- prettier-ignore -->
{{ $lang := .Get 0 -}}
{{ $Lang := $lang | humanize -}}
{{ $howMany := .Get 1 | default 10 -}}
{{ $adopters := where $.Site.Data.ecosystem.adopters ".components" "intersect" (slice $Lang) -}}

## Who's using OpenTelemetry {{ $Lang }}?

OpenTelemetry {{ $Lang }} is in use by a number of organizations, including:

{{ range first $howMany (sort $adopters "name") -}}

<!-- prettier-ignore -->
- [{{ .name }}]({{ .url }})
{{- end }}

For the complete list, and instructions for adding your organization, see
[Adopters](/ecosystem/adopters/).
```

IMHO, our YAML datafile should not use "humanized" component names, but we can address that later.